src/routes/feedRoute.js

```diff
@@ -27,6 +29,9 @@ router.post('/get-activity-by-id', getActivityById);
 router.post('/remove-activity-by-id', removeActivityById);
 router.post('/sync-user-score', syncUserScore);
 
+router.post('/check-activity-score', activityScore);
+router.get('/get-weight-value', getWeightValue);
+
```

Comment:
mas @n-permana can we somehow add guard to these routes?
Let's say just a password just to make sure that these routes can't be accessed by unathorized person

Reply:
Ok, I will add that auth to those endpoint